src/grt/src/GlobalRouter.cpp

```diff
@@ -429,6 +429,8 @@ NetRouteMap GlobalRouter::findRouting(std::vector<Net*>& nets,
 {
   NetRouteMap routes;
   if (!nets.empty()) {
+    MakeWireParasitics builder(logger_, resizer_, sta_, db_->getTech(), this);
+    fastroute_->setMakeWireParasiticsBuilder(&builder);
     routes = fastroute_->run();
```

Comment:
It would be good to "fastroute_->setMakeWireParasiticsBuilder(nullptr)" after run() to ensure the builder isn't used after it goes out of scope.